src/style/app.module.css

```diff
@@ -2149,8 +2874,8 @@ input[type='radio']:checked + label:hover {
 .pluginStoreBtn {
   width: 100%;
   background-color: var(--bs-white);
-  color: var(--brown-color);
-  border: 0.5px solid var(--brown-color);
+  color: var(--dark-grey-font);
+  border: 0.5px solid var(--dark-grey-border);
```

Comment:
Use variable names that describe the function of the color and not the color itself. This will help with adding a Dark Mode feature.
- For example `--search-button-bg` is much better than `--light-blue`